Package.swift

```diff
@@ -1,26 +1,41 @@
-// swift-tools-version:5.2
-// The swift-tools-version declares the minimum version of Swift required to build this package.
+// swift-tools-version:5.0
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
 
 import PackageDescription
 
-let sources = ["", "Helpers", "HttpClient", "HttpClient/Util", "Internal", "Models", "Utils", "Vendor/Reachability"]
 let package = Package(
     name: "WindowsAzureMessaging",
     platforms: [
-        .iOS(.v10)
+    platforms: [
+        .iOS(.v9),
+        .macOS(.v10_10),
+        .tvOS(.v11)
     ],
     products: [
         .library(
             name: "WindowsAzureMessaging",
+            type: .static,
             targets: ["WindowsAzureMessaging"]),
     ],
     dependencies: [],
     targets: [
         .target(
             name: "WindowsAzureMessaging",
-            path: "src/WindowsAzureMessaging/WindowsAzureMessaging",
-            sources: sources,
-            cSettings: sources.map { CSetting.headerSearchPath($0) }
+            path: "WindowsAzureMessaging/WindowsAzureMessaging",
```

Comment:
AppCenter SDK has [inclide/*.h](https://github.com/microsoft/appcenter-sdk-apple/tree/3.3.0/AppCenter/AppCenter/include) links to public headers. Does this work correctly without similar thing?

Reply:
No, I'll have to do that for Swift Package Manager to work